en/docs/learn/message-transformation.md

```diff
@@ -37,66 +36,39 @@ This guide explains how to create a simple integration to convert a JSON payload
 
-   
-### Step 5: Error handling
-1. Click on the ➕ button again and select **`If`** from the node panel.
-2. Enter the condition as `xmlData is xml` and press **`Save`**.
-   <a href="{{base_path}}/assets/img/message-transformation/add-if.png"><img src="{{base_path}}/assets/img/message-transformation/add-if.png"" alt="Add If" width="70%"></a>
-3. Click on the `If` condition true(`xmlData is xml`)  path ➕ sign and add a **`Return`** from the node panel.
-4. Select the `xmlData` variable from the dropdown and click **`Save`**.
-   <a href="{{base_path}}/assets/img/message-transformation/add-return.png"><img src="{{base_path}}/assets/img/message-transformation/add-return.png"" alt="Add Return" width="70%"></a>
-5. Click on the `If` condition `Else` path ➕ sign and add another **`Return`** from the node panel.  
-6. Enter `http:BAD_REQUEST;` as the value and click **`Save`**.  
-   <a href="{{base_path}}/assets/img/message-transformation/add-else-return.png"><img src="{{base_path}}/assets/img/message-transformation/add-else-return.png"" alt="Add Else Return" width="70%"></a>
-7. The final design will look like below.             
-   <a href="{{base_path}}/assets/img/message-transformation/final-design.png"><img src="{{base_path}}/assets/img/message-transformation/final-design.png"" alt="Final Design" width="70%"></a>
-8. The final code will look like below. The source view can be accessed by clicking on the `</>` button in the top right corner.
-   ```ballerina
-   import ballerina/http;
-   import ballerina/xmldata;
-   
-   service /convert on new http:Listener(9090) {
-   
-       function init() returns error? {
-       }
-   
-       resource function post toXml(@http:Payload json input) returns xml|http:InternalServerError|http:BadRequest {
-           do {
-   
-               xml? xmlData = check xmldata:fromJson(input);
-               if xmlData is xml {
-                   return xmlData;
-               } else {
-                   return http:BAD_REQUEST;
-               }
-   
-           } on fail error e {
-               return http:INTERNAL_SERVER_ERROR;
-           }
-       }
-   }
 
-   ```
-
-### Step 6: Run the integration
+### Step 5: Run the integration
 1. Click on the **`Run`** on the run button in the top right corner to run the integration.
 2. The integration will start and the service will be available at `http://localhost:9090/convert`.
-3. The service can be tested using a tool like [Postman](https://www.postman.com/) or [curl](https://curl.se/) by sending a POST request with a JSON payload to the service endpoint.
-   ```curl
-   curl -X POST "http://localhost:9090/convert/toXml" -H "Content-Type: application/json" -d '{"name":"John", "age":30, "car":"Honda"}'
-   ```
-4. The response will be an XML representation of the JSON payload.  
-   ``` 
+3. Click on the **`Try it`** button to open the embedded HTTP client.
```

Comment:
can we be more specific in here? we can suggest to use the below auto popup after the running integration

![Uploading Screenshot 2025-03-14 at 09.17.49.png…]()

Reply:
Will update with another PR with screenshots